test/Elastica/DocumentTest.php

```diff
@@ -85,7 +85,7 @@ public function testHasId()
     {
         $document = new Document();
         $this->assertFalse($document->hasId());
-        $document->setId('');
+        $document->setId(null);
```

Comment:
@thePanz I really wonder why / how this was passing before 🤔

Reply:
Not sure, that's weird!